data/json/itemgroups/mall_item_groups.json

```diff
@@ -140,6 +140,28 @@
     "type": "item_group",
     "items": [ [ "bra", 70 ], [ "panties", 70 ], [ "boy_shorts", 50 ], [ "boxer_shorts", 70 ], [ "briefs", 50 ] ]
   },
+  {
+    "id": "underwear",
```

Comment:
This should probably just go in clothing.json, and if we don't already have that we should make it. There's nothing specifically mallish about it.

Reply:
yeah I didn't want to edit a bunch of different files when this was already so large for the reviews, but it can certainly be moved.